charts/portworx/templates/hooks/pre-upgrade/retain-daemonset-install.yaml

```diff
@@ -42,8 +35,7 @@ spec:
           command: ['/bin/sh',
                     '-c',
                     'kubectl -n kube-system annotate DaemonSet portworx-api helm.sh/resource-policy=keep --overwrite;
-                     kubectl -n kube-system annotate DaemonSet portworx helm.sh/resource-policy=keep --overwrite;
-
+                     kubectl -n kube-system annotate DaemonSet portworx helm.sh/resource-policy=keep --overwrite || true;
```

Comment:
Also should we be concerned that the helm resource-policy label will be added to even the operator created objects like portworx-api, stork, etc? Hopefully it should not be a problem, but just check it out.

Reply:
yes, that's should be ok.